cmake/RTGen.cmake

```diff
@@ -94,9 +94,11 @@ function(_rtgen_interface FILENAME OUTFILES_VAR)
             # If generating bindings set the output directory to "bindings/{LANG}"
             # Create the directory if it doesn't exist yet
             set(RTGEN_OUTPUT_DIR "${CURR_BINDINGS_DIR}/${LANG}")
-            if (NOT EXISTS RTGEN_OUTPUT_DIR)
-                opendaq_create_dir(${RTGEN_OUTPUT_DIR})
-            endif()
+        endif()
+
+        # Create the directory if it doesn't exist yet
+        if (NOT EXISTS RTGEN_OUTPUT_DIR)
```

Comment:
I think you're fixing the symptom not the cause. 

The C++ bindings directory should always be there as it is the same as `${CMAKE_CURRENT_BINDARY_DIR}`.
As the comment above says all other bindings go into `${CMAKE_CURRENT_BINDARY_DIR}/bindings/<lang>` and as this directory is not created by CMake it is created by RTGen script. 

If you redirect `RTGEN_OUTPUT_DIR` to somewhere else its on you to make sure it exists. I don't really think it is the job of RTGen to do this as it is an early sign that you're doing something wrong if redirecting was not intended.